Contactpage/validation message js

ReposByHussam opened the pull request:
This PR closes issue. #69 

I have already done the validation messages on the Contactpage on another issue, so this is a fix of multiple things that i have added in this card. First is the change of background on the contactpage so it matches the other pages:

<img width="1544" height="1273" alt="image" src="https://github.com/user-attachments/assets/273aac0a-0365-4903-9882-59253592d5fc" />

Secondly, i changed the button layout to match the behaviour and look of the other pages that are on dark - layouts.

I also changed the contact-information to match the footer on the rest of the pages: 

The footer is however redesigned on the contact page to avoid any duplication that may not be appealing to the eye.
 
And lastly, i linked the footer as well as changed the name to "kontakta oss", by clicking on it, the user is directed to the contact page.
<img width="1374" height="277" alt="image" src="https://github.com/user-attachments/assets/a0163bad-966b-478e-9462-4853a8141687" />

leolinder-hash reviewed (commented):
Hey! I think that all the changes that you've made look great! The only thing I noticed is that there is a conflict with index.html. It's probably that we added a link to the eventpage that you might not have in your version. When this is fixed I'll approve straight away!

Great work :)

ReposByHussam commented:
> Hey! I think that all the changes that you've made look great! The only thing I noticed is that there is a conflict with index.html. It's probably that we added a link to the eventpage that you might not have in your version. When this is fixed I'll approve straight away!
> 
> Great work :)

Thank you for the great feedback! I have now resolved the issue and updated my version so that is routes to eventPages.html!
